bootstrap/components/topology/storm-topology-component.json

```diff
@@ -55,6 +55,40 @@
             "defaultValue": "hdfs://localhost:9000/tmp/hbase"
           }
         ]
+      },
+      {
```

Comment:
It's always shown since field list are not dynamic. 
And yes, it's optional. Users don't need to add the config if topology doesn't contain secured cluster or auto token delegation is not needed.